portfolio/src/main/webapp/index.html

```diff
@@ -43,9 +40,49 @@ <h3 id="subhead"> &#9830; Chiamaka  (Chee-uh-mok-uh)  Chukwuka  (Chew-kwoo-kah).
       <p> 
       </p>
     </div>
+        <!--
         <p>Click here to see a greeting and an Array</p>
         <button onclick="getText()">Fetch Text!</button>
+        <br/><br/>
         <div id="text-container"></div>
+        -->
+        <br/><br/>
+        <br/><br/>
+        <br/><br/>
+
+        <form action="/data" method="POST">
+
+        <p>Comments:</p>
+
+            <input type="text" name="input-string">
+            <input type="submit" />
+
+        </form>
+        <br/><br/>
+
+        <form>
+
+        <p>Enter a maximum number of comments:</p>
+
+            <input type = "number" name="max" id = "max">
+            
+
+        </form>
+
+        <br/><br/>
+
+        <button onclick="getComments()"> Refresh Comments </button>
```

Comment:
You can make this smoother this by adding an onchange event handler to your number input above.  That way the comments are automatically refreshed when the max is changed

Reply:
resolving because I added onchange event